packages/injected/src/utilityScript.ts

```diff
@@ -35,18 +30,12 @@ export type Builtins = {
   requestAnimationFrame: Window['requestAnimationFrame'],
   cancelAnimationFrame: Window['cancelAnimationFrame'],
   requestIdleCallback: Window['requestIdleCallback'],
-  cancelIdleCallback: (id: number) => void,
+  cancelIdleCallback: Window['cancelIdleCallback'],
   performance: Window['performance'],
   // eslint-disable-next-line no-restricted-globals
-  eval: typeof window['eval'],
-  // eslint-disable-next-line no-restricted-globals
   Intl: typeof window['Intl'],
   // eslint-disable-next-line no-restricted-globals
   Date: typeof window['Date'],
-  // eslint-disable-next-line no-restricted-globals
-  Map: typeof window['Map'],
```

Comment:
Why did they go away?

Reply:
We are not protecting against Map/Set tampering anymore. Let me explain that in the description.